### DISCUSSION TITLE
Admin Endpoint for Role Usage

### DISCUSSION THREAD

**Discussion by simon-leech** [GEOLYTIX/xyz#1755] [Ideas]
## Description

Would it be possible to write an endpoint that an admin could hit that would provide a json object of which layers are accessible for which roles within a locale?
It would be useful in xyz - as it gives the admin the ability to quickly find out what each role does (layer access / locale access for example). 

Opening an RFC to talk about this

Would potentially get tricky with roles defined within layers that have ability to show / hide different fields, draw / delete etc.

**Use Case**
I often get asked what can role 'A' see / do
So would be ideal if there was a way for an admin to be given a table ideally of roles and their abilities

## Type of Issue 
Please delete options that are not relevant, and select all options that apply. 

- ✅ New Feature

**Comment by dbauszus-glx** [comment]
Yes. This is very easily achievable with a one liner. I created an issue for this. https://github.com/GEOLYTIX/xyz/issues/1800
